Source/WebKit/WebProcess/Plugins/PluginView.cpp

```diff
@@ -595,6 +595,22 @@ bool PluginView::findString(const String& target, WebCore::FindOptions options,
     return m_plugin->findString(target, options, maxMatchCount);
 }
 
+Vector<FloatRect> PluginView::rectsForTextMatchesInRect(const IntRect& clipRect) const
+{
+    if (!m_isInitialized)
+        return { };
+
+    return m_plugin->rectsForTextMatchesInRect(clipRect);
```

Comment:
Please protect `m_plugin`. Ideally, you'd add a `protectedPlugin()` to this class which returns a `Ref<PDFPluginBase>` and call this here.

Reply:
Iiiiiiiiii think maybe we should do this whole mechanism in one patch? I can look into that.